munininfluxdb/commands/dump.py

```diff
@@ -0,0 +1,80 @@
+import logging
```

Comment:
This is one of the subcommands of the main command. It is loaded in `munininfluxdb/main.py`.